$NetBSD: patch-aa,v 1.3 2006/10/02 21:34:28 rillig Exp $

If LWP::UserAgent is not available, exit before the module is required
implicitly by Bio::DB::GDB.

--- t/GDB.t.orig	2001-03-01 10:11:15.000000000 +0100
+++ t/GDB.t	2006-10-02 23:30:15.000000000 +0200
@@ -33,10 +33,9 @@ BEGIN {
 	foreach ( 1..$NUMTESTS) { skip(1,1); }
 	$error = 1;
     }
-}
-
-if( $error == 1 ) {
-    exit(0);
+    if( $error == 1 ) {
+	exit(0);
+    }
 }
 
 use Bio::DB::GDB;
